vms/platformvm/vm.go

```diff
@@ -247,6 +252,8 @@ func (vm *VM) Initialize(
 		return err
 	}
 
+	go vm.startMempoolPruner(mempoolPrunerFrequency)
```

Comment:
Should we comment that we don't `vm.awaitShutdown.Add(1)` here because the mempool pruner grabs the context lock?